elements/bulbs-reading-list/components/bulbs-reading-list-item.js

```diff
@@ -3,13 +3,149 @@ import {
   BulbsHTMLElement,
 } from 'bulbs-elements/register';
 import invariant from 'invariant';
+import {
+  debouncePerFrame,
+  filterBadResponse,
+  getResponseText,
+  getAnalyticsManager,
+  InViewMonitor,
+  prepGaEventTracker,
+} from 'bulbs-elements/util';
+
+let pageStartDebouncer = debouncePerFrame();
 
 class BulbsReadingListItem extends BulbsHTMLElement {
   attachedCallback () {
-    invariant(this.dataset.id, '<bulbs-reading-list-menu> requires a "data-id" attribute');
-    invariant(this.dataset.href, '<bulbs-reading-list-menu> requires a "data-href" attribute');
-    invariant(this.dataset.partialUrl, '<bulbs-reading-list-menu> requires a "data-partial-url" attribute');
-    invariant(this.dataset.title, '<bulbs-reading-list-menu> requires a "data-title" attribute');
+    const markdownText = '<bulbs-reading-list-item> requires attribute: ';
```

Comment:
thought is to add a `is-loaded` attribute here so the template [here](https://github.com/theonion/omni/blob/master/avclub/avclub/templates/core/detail/_base.html#L64) doesn't reinitialize it